maintenance/wikia/importMessagingWiki.php

```diff
@@ -0,0 +1,195 @@
+<?php
+require_once __DIR__ . '/../Maintenance.php';
+
+class ImportMessagingWiki extends Maintenance {
+
+	/** @var string MESSAGING_WIKI_DB messaging wiki database name */
+	const MESSAGING_WIKI_DB = 'messaging';
+
+	/**
+	 * @var array Array of messages from messaging wiki (same format as in i18n php files
+	 */
+	protected $messages = [];
+
+	/**
+	 * @var bool Whether to modify source files in app repo or write to tmp
+	 */
+	protected $isDryRun = true;
+
+	public function __construct() {
+		parent::__construct();
+
+		$this->addOption( 'dry-run', 'Don\'t make any changes to i18n files - print output in /tmp/ folder.' );
+		$this->addOption( 'message', 'Message name to process', false, true /* $withArg */ );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "message" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)